distributed/worker.py

```diff
@@ -486,7 +486,7 @@ def compute_one(self, data, key=None, function=None, args=None, kwargs=None,
                     "Function: %s\n"
                     "args:     %s\n"
                     "kwargs:   %s\n",
-                    str(funcname(function))[:1000], 
```

Comment:
Those trailing spaces were fixed by my editor. I can re-edit this commit to avoid trimming them if you prefer.